src/components/SegmentedRadioGroup/__stories__/SegmentedRadioGroup.stories.tsx

```diff
@@ -78,6 +79,19 @@ export const Disabled: Story = {
     },
 };
 
+export const ChildrenMap: Story = {
```

Comment:
Let's call the story `WithTooltip` and wrap every option in tooltip. The story should demonstrate how it looks, not how it works inside